script/packtpub.py

```diff
@@ -123,10 +123,10 @@ def download_ebooks(self, types):
             filename=self.info['filename'] + '.' + type)
             for type in types]
 
-        directory = self.__config.get('path', 'path.ebooks')
-
-        if self.__config.get('path', 'path.separate') == "true":
-            directory = directory + '/' + self.info['title'] + ' - ' + self.info['author']
+        if self.__config.has_option('path', 'path.separate.extras'):
+            directory = self.__config.get('path', 'path.separate.ebooks') + '/' + self.info['title'] + ' - ' + self.info['author']
```

Comment:
I'll wait to merge the request when you will add the sanity (regexp) check and the `path.separate.space`.
Maybe something like this should be already enough
```
.encode('ascii', 'ignore').replace(' ', '_')
```